docs/project/about.md

```diff
@@ -59,15 +59,39 @@ list from there with "The Pyodide development team" like in the example below:
 - Stack Overflow: [stackoverflow.com/questions/tagged/pyodide](https://stackoverflow.com/questions/tagged/pyodide)
 - Discord: [Pyodide Discord](https://dsc.gg/pyodide)
 
-## Donations
+## Institutional and Financial Support
 
-We accept donations to the Pyodide project at
-[opencollective.com/pyodide](https://opencollective.com/pyodide). All donations
-are processed by the [Open Source Collective](https://www.oscollective.org/) -- a
-nonprofit organization that acts as our fiscal host.
+Pyodide is a community-driven project that has benefited from the support of various institutions, grants, and employers who have allocated resources and time to its development.
 
-Funds will be mostly spent to organize in-person code sprints and to cover
-infrastructure costs for distributing packages built with Pyodide.
+### Current Support
```

Comment:
Let's merge the current support and past support sections. If the end date of the period is "present" then it's current support otherwise it's past support.